_resources/port1.0/group/common_lisp-1.0.tcl

```diff
@@ -75,8 +75,12 @@ proc common_lisp::add_dependencies {} {
     }
 
     if {[option common_lisp.ecl]} {
-        depends_build-delete    port:ecl
-        depends_build-append    port:ecl
+        depends_build-delete    port:ecl \
+                                port:ecl-devel \
+                                path:bin/ecl:ecl \
+                                path:bin/ecl:ecl-devel
+
+        depends_build-append    path:bin/ecl:ecl
```

Comment:
@mascguy I not sure how to make it cleaner. Do you have any suggestion?

Reply:
> @mascguy I not sure how to make it cleaner. Do you have any suggestion?

Are you referring to the combinations of `ecl` and `ecl-devel`, which you're deleting from `depends_build`?

Reply:
@mascguy yes, I'm

Reply:
I think it's fine as-is.